amazon-s3/5.6/modules/ROOT/pages/amazon-s3-connector-config-topics.adoc

```diff
@@ -9,6 +9,151 @@ In the `Create Object` operation, set the `Content Length` to a value greater th
 
 To encrypt objects that you want to store in S3 buckets using customer managed master keys, specify a Customer Master Key ID in the `KMS Master Key` field in the `Create Object` configuration.
```

Comment:
```suggestion
To encrypt objects that you want to store in S3 buckets using customer-managed master keys, specify a Customer Master Key ID in the `KMS Master Key` field in the `Create Object` configuration.
```

Reply:
How do creating an object operation and master key--the main tasks of this topic set, according to the Next Steps text, specifically relate to the following subtopics?
It wasn't until I got to the final sentence that I realized that the two tasks here were
1. Configure master key
2. Provide credentials
Should those be explicitly stated using subheadings, and the topic arranged around this?

Reply:
Please ignore edits below that reveal that I did not understand the focus of this topic when I first edited it; I was too lazy to remove them, and maybe some make sense.